classes.js

```diff
@@ -0,0 +1,74 @@
+const listBooks = document.querySelector('.book-list');
+const form = document.querySelector('.form-input');
+const [title, author] = form.elements;
+
+const inputBook = {};
+let books = new Array([]);
+
+if (localStorage.savedBooks) {
+  books = JSON.parse(localStorage.getItem('savedBooks'));
+}
+
+title.addEventListener('change', () => {
+  inputBook.title = title.value;
+});
+
+author.addEventListener('change', () => {
+  inputBook.author = author.value;
+});
+
+const populateFields = () => {
+  localStorage.setItem('savedBooks', JSON.stringify(books));
+};
+
+const Book = class {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  static removeBook(book) {
+    const result = books.filter((b) => b !== book);
+    books = result;
+    populateFields();
+  }
+
+  static addBook = (newBook) => {
+    books.push(newBook);
+    populateFields();
+    this.displayBooks();
+  };
+
+  static displayBooks = () => {
+    listBooks.innerHTML = '';
+    books.map((book) => {
+      const bookDiv = document.createElement('tr');
+      const elementBook = document.createElement('td');
+      const deleteBtn = document.createElement('button');
+      deleteBtn.textContent = 'Remove';
+
+      elementBook.textContent = `"${book.title}" by ${book.author}`;
+
+      bookDiv.classList.add('container-book');
+      bookDiv.appendChild(elementBook);
+      bookDiv.appendChild(deleteBtn);
+
+      listBooks.appendChild(bookDiv);
+
+      deleteBtn.addEventListener('click', () => {
+        this.removeBook(book);
+        listBooks.removeChild(bookDiv);
+      });
+      return listBooks;
+    });
+  };
+};
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  Book.addBook(new Book(inputBook.title, inputBook.author));
+  form.submit();
```

Comment:
- [ ] **[OPTIONAL]** I think it is somewhat unnecessary to submit the form here. On line 68 you prevent submission so you can add a book. After adding a book, I don't think there is any need to submit the form again since you have accomplished what you want to do which is to add a book. I would recommend you remove line 70 to avoid unnecessary page refresh.